lib/core/src/client/preview/url.test.ts

```diff
@@ -52,39 +46,6 @@ describe('url', () => {
     });
   });
 
-  describe('getIdFromLegacyQuery', () => {
-    const getRawStory = () => {};
-    const store: unknown = { getRawStory };
-    it('should parse story paths', () => {
-      expect(getIdFromLegacyQuery({ path: '/story/story--id' }, store as StoryStore)).toBe(
-        'story--id'
-      );
-    });
-    it('should use legacy parameters to look up custom story ids', () => {
-      const customStore: unknown = {
-        getRawStory: () => ({ id: 'custom--id' }),
-      };
-      expect(
-        getIdFromLegacyQuery(
-          { selectedKind: 'kind', selectedStory: 'story' },
-          customStore as StoryStore
-        )
-      ).toBe('custom--id');
-    });
-    it('should use fall-back behavior for legacy queries for unknown stories', () => {
-      expect(
-        getIdFromLegacyQuery(
-          { path: null, selectedKind: 'kind', selectedStory: 'story' },
-          store as StoryStore
-        )
-      ).toBe('kind--story');
-    });
-
-    it('should not parse non-queries', () => {
-      expect(getIdFromLegacyQuery({}, store as StoryStore)).toBeUndefined();
-    });
-  });
```

Comment:
So glad to see this code go!

Reply:
Well actually it isn't really gone: https://github.com/storybookjs/storybook/pull/11111